### DISCUSSION TITLE
I have not an education email

### DISCUSSION THREAD

**Discussion by Autooo13** [community/community#58911] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am sorry to say that students in our school do not have education email, so I can only use my own email to communicate.So I can't pass copilot student certification when I apply for it. What can I do?

**Comment by SUBHADIPMAITI-DEV** [comment]
Contact the certification provider: Reach out to the organization or platform that offers the Copilot student certification. Explain your situation and inquire if they have alternative methods or accommodations for students who don't have access to an educational email address. They may be able to provide guidance or offer an alternative solution.

Use your personal email with an explanation: While it may not meet the specific requirements, you can still try using your personal email to apply for the certification. In your application, include a clear and detailed explanation that you don't have an educational email and provide the reasons why (e.g., your school doesn't provide them). Mention your enthusiasm for the certification and your commitment to learning. The certification provider may take your circumstances into consideration.

Explore other certification options: If the Copilot student certification is not feasible due to the educational email requirement, consider exploring alternative certifications or training programs that don't have this limitation. There are numerous other certifications available in the field of programming and software development that can help enhance your skills and knowledge.

Utilize educational resources: While certification can be valuable, it's important to remember that it's not the only path to learning and growth. Focus on utilizing educational resources that are available to you, such as online tutorials, coding platforms, open-source projects, and interactive learning websites. These resources can provide practical experience and help you build a strong foundation in programming.
